test/it/findutils/find_test.sh

```
@@ -17,3 +17,9 @@ TestFindUnknown() {
     export TEST_DIR=/tmp/mimixbox/it/find
     find ${TEST_DIR} -bogus
 }
+TestFindHelp() {
+    find --help
+}
+TestFindVersion() {
+    find --version
```

Shellcheck (0.11.0)

[info] Some finds don't have a default path. Specify '.' explicitly.

(SC2185)